langlib/langlib-test/src/test/resources/test-src/type-param/type_param_bound_type_as_cet.bal

```diff
@@ -184,7 +184,7 @@ function assertTrue(any|error actual) {
 }
 
 function assertValueEquality(anydata|error expected, anydata|error actual) {
-    if expected == actual {
+    if isEqual(expected, actual) {
```

Comment:
Shouldn't the arguments be switched? `isEqual` seems to have `actual` as the first param?

Reply:
Applicable for some other functions too.

Reply:
It doesn't matter here right, it says if first argument equals to second argument do this.

Reply:
True that the order doesn't matter in terms of the logic, but isn't it misleading when you consider the parameter names? IMO we should either fix the order here to be in-line with `isEqual` or update the signature of `isEqual` to have names that imply a generic value comparison, maybe `v1` and `v2`.

Reply:
`isEqual(expected, actual)` misleading in to what?

Reply:
The fact that we're passing `expected` as the first argument and `actual` as the second argument to a function that has the inverse as the signature.

```ballerina
isolated function isEqual(anydata|error actual, anydata|error expected) returns boolean {
    return actual is anydata && expected is anydata && actual == expected || actual === expected;
}
```

Reply:
Misleading is when it says one thing and does some other thing, this says that it checks if two args are equal and it does exactly that. 
In my view what variable it passes to what parameter is just a preference (talking in this context)

Reply:
What I found potentially misleading was that we first pass `expected` and then `actual` to a function that seems to accept `actual` as the first parameter and `expected` as the second - not the implementation itself.

`isEqual` currently says it takes an `actual` value and an `expected` value and compares the two to see if they are equal, right? While highly unlikely in this scenario, what if we eventually add like an actual vs expected log there, given that it is an arbitrary comparison function?

Why can't we change the parameter names of the function to reflect that it's a generic comparison of two values, if the order cannot be changed?

Reply:
We can absolutely change the param names.

If we decide to log something from a isolated function called `isEqual` that returns and boolean I think that function signature is misleading.

Reply:
+1, seems to be the case in several other files too.